eth-contracts/contracts/service/DelegateManager.sol

```diff
@@ -144,7 +148,21 @@ contract DelegateManager is RegistryContract {
         return delegateInfo[delegator][_target];
     }
 
-    function makeClaim() external {
+    // Distribute proceeds of reward
+    function claimRewards() external {
```

Comment:
TODO: See if its worth splitting processClaim into a separate tx? 
          Primary concern is around gas consumption...
          This tx ends up minting tokens, transferring to staking, and doing below updates
          Can be stress tested and split out if needed